filters/auth/authclient.go

```diff
@@ -30,6 +30,12 @@ type authClient struct {
 	cli *net.Client
 }
 
+type tokeninfoClient interface {
+	getTokeninfo(token string, ctx filters.FilterContext) (map[string]any, error)
```

Comment:
If we have `any` here the PR should also change the definition of `func (ac *authClient) getTokeninfo(token string, ctx filters.FilterContext) (map[string]interface{}, error) ` I think.

Reply:
I guess we need to create a ticket to change `interface{}` to `any` across the board https://stackoverflow.com/questions/71964357/what-is-the-any-type-in-go-1-18

Reply:
> Another maybe more important question would be if we can re-use it for other cases like tokenintrospection, which serves a similar purpose.

It should be possible but I think it makes sense to get experience using it with tokeninfo first.